source/includes/steps-tutorial-backend.yaml

```
@@ -114,14 +126,16 @@ content: |
           timestamp: new Date()
         }
         // insert a document into your collection
-        const insertDoc = await context.services.get("<clusterName>").db("<dbName>").collection("<collectionName>").insertOne(issueDoc);
+        const insertDoc = await context.services.get("mongodb-atlas").db("tutorials").collection("myGithubIss").insertOne(issueDoc);
```

To match the other tutorial apps: (+ Nitpick: what is "Iss"?)
```suggestion
        const insertDoc = await context.services.get("mongodb-atlas").db("tracker").collection("myGithubIss").insertOne(issueDoc);
```